src/selenium/trackPage.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable no-empty-label */
+'use strict';
+
+const BasePage = require('./basePage.js');
+const By = require('selenium-webdriver').By;
+
+const TrackPage = Object.create(BasePage);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'const' statement. (one-var)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)